# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gnustep-base/hotfix-libxml.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

From 37913d006d96a6bdcb963f4ca4889888dcce6094 Mon Sep 17 00:00:00 2001
From: Frederik Seiffert <frederik@algoriddim.com>
Date: Wed, 3 May 2023 13:49:09 +0200
Subject: [PATCH] Fix GSXML compatibility with libxml2 v2.11.0

Fixes #294.
---
 Source/Additions/GSXML.m | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Source/Additions/GSXML.m b/Source/Additions/GSXML.m
index 37f509564a..01a24c5aa1 100644
--- a/Source/Additions/GSXML.m
+++ b/Source/Additions/GSXML.m
@@ -2671,10 +2671,12 @@ + (void) initialize
                   return NULL;
                 }
               ret->owner = 1;
+#if LIBXML_VERSION < 21100
               if (ret->checked == 0)
                 {
                   ret->checked = 1;
                 }
+#endif
             }
         }
     }
